src/trace/dma-trace.c

```diff
@@ -224,9 +224,9 @@ static int dma_trace_buffer_init(struct dma_trace_data *d)
 	void *buf;
 	unsigned int flags;
 
-	/* allocate new buffer */
```

Comment:
Is this DMA hardware specific? Commit message doesn't really explains why we need to align to 128 byte.

Reply:
Yes it is a DMA driver specific requirement. The constant is defined there hda-dma.c (see the second patch). I could not come up with any clean way to bring driver side constant to dtrace code and it hardly causes any harm to ask for 128-aligned buffer for platforms that do not need it. By default all memory allocations are 64-byte aligned.

Reply:
We should do this as a platform config macro. This also will be needed in Zephyr per DMA driver, so method should be compatible.

Reply:
@jsarha please modify DMA_TRACE_LOCAL_SIZE in sof/src/platform/tigerlake/include/platform/platform.h and friends. This is platform specific and only used for trace, so will be very localized.